src/Lykke.Snow.AuditService/Settings/AppSettings.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) 2023 Lykke Corp.
+// See the LICENSE file in the project root for more information.
+
+namespace Lykke.Snow.AuditService.Settings
+{
+    public class AppSettings
+    {
+        public AuditServiceSettings AuditService { get; set; } = null!;
```

Comment:
You've enabled "Nullable" feature for the project and at the same time assign null value to a non-nullable property. Why is that?
```suggestion
        public AuditServiceSettings? AuditService { get; set; } = null;
```